packages/metro-resolver/src/__tests__/index-test.js

```diff
@@ -56,9 +56,19 @@ const CONTEXT: ResolutionContext = (() => {
       },
       'other-root': {
         node_modules: {
+          'banana-module': {
+            'package.json': true,
+            'main.js': true,
+          },
           banana: {
             'package.json': true,
             'main.js': true,
+            node_modules: {
+              'banana-module': {
+                'package.json': true,
+                'main.js': true,
+              },
```

Comment:
I like the example when you suggested a regular module under `banana/node_modules/other-module/main.js` that can also be resolved from `banana/main.js`.